Publisher/nl/productupdate20230511.md

```diff
@@ -0,0 +1,35 @@
+# Productupdate - Opvolgactie verbeteringen, uitschrijfbevestiging en verzendmomenten uitsluiten met RRule
+
+## Opvolgactie op basis van veldwijziging
+Het is nu mogelijk om in het _bestemming checken_ blok te kijken of een veld is aangepast. Hierbij kan je aangeven dat de opvolgactie uitgevoerd moet worden als het veld naar een specifieke waarde is aangepast of bij iedere wijziging waarbij de waarde niet gespecificeerd is. 
+
+Na het toevoegen van het blok, kies je bij _vergelijking_ voor _is bijgewerkt_ voor iedere wijziging of voor _is bijgewerkt naar_ voor een specifieke wijziging. Bij deze laatste optie krijg je een invoerveld om de waarde te specificeren.
+
+## Opvolgactie op basis van een specifieke foutmelding
+Als je gebruik maakt van een opvolgactie waarbij de trigger een _e-mailfout_ is, heb je nu de mogelijkheid om een _check fout_ blok toe te voegen. In dit blok kun je aangeven bij welke foutmelding de opvolgactie moet worden uitgevoerd. 
+
+Naast een specifieke foutcode zijn ook de opties 'foutmelding die bij opnieuw versturen misschien niet meer optreedt' en 'foutmelding die bij een volgende verzending mogelijk weer optreedt' beschikbaar. 
+
+## Uitschrijfbevestiging bij gebruik {unsubscribe}-tag
+Bij het gebruik van de [{unsubscribe}](https://www.copernica.com/nl/documentation/email-editor-unsubscribelink-webversion)-tag kun je nu aangeven of de uitschrijving bevestigd moet worden. Op de uitschrijfpagina krijgt de ontvanger de vraag "Weet je zeker dat je je wilt afmelden?" met een button om dit te bevestigen. Dit is toegevoegd om te voorkomen dat spamfilters, die links bezoeken uit je mail, ervoor zorgen dat je ontvangers direct worden uitgeschreven.
```

Comment:
Bij het gebruik van de [{unsubscribe}](https://www.copernica.com/nl/documentation/email-editor-unsubscribelink-webversion)-tag kun je nu aangeven of de uitschrijving bevestigd moet worden. Wanneer je deze optie activeert, krijgt de ontvanger bij het openen van de uitschrijfpagina de vraag "Weet je zeker dat je je wilt afmelden?" met een button om de uitschrijving te bevestigen. Dit is gedaan om te voorkomen dat ontvangers onbedoeld worden uitgeschreven als gevolg van spamfilters die links uit je e-mail volgen.